package.json

```diff
@@ -62,6 +63,7 @@
     "lob": "^6.6.3",
     "objection": "^3.0.1",
     "pg": "^8.7.3",
+    "redis": "^4.3.1",
```

Comment:
Based on my understanding, we are currently using a customized class for globally storing key-value pairs for caching as of now. Does that mean Redis is a direct dependency in our application (i.e. are we running a Redis server in parallel?)?

If you added it while testing, try removing Redis and running the tests again. If they pass, we'll not need to have Redis as a dependency.

Reply:
I added Redis to test my options regarding custom storage. I have removed the leftover.